ftw_tools/cli.py

```diff
@@ -615,9 +615,9 @@ def inference_download(
 @click.option(
     "--model",
     "-m",
-    type=click.Choice(list(MODEL_REGISTRY.keys())),
+    type=str,
```

Comment:
This is a bit unfortunate because it was quite convenient to get autocomplete on the names. Can we restore this and still allow arbitrary inputs?